src/app/dataset-view/dataset.component.ts

```diff
@@ -147,33 +160,35 @@ export class DatasetComponent
 
     private initOverviewTab(): void {
         this.datasetViewType = DatasetViewTypeEnum.overview;
-        this.appDatasetService.getDatasetOverview(this.getDatasetId());
+        this.appDatasetService.getDatasetOverview(this.datasetInfo);
```

Comment:
In all functions like this, please pass `datasetInfo` as a parameter of the function